style.css

```diff
@@ -640,6 +630,68 @@ input:valid {
   text-align: left;
 }
 
+.btn-live {
+  width: 115px;
+  margin-right: 18px;
+  font-size: 17px;
+  font-weight: 500;
+  padding-left: 0;
+  display: flex;
+  text-decoration: none;
+  justify-content: space-between;
+}
+
+.btn-source {
+  width: 156px;
+  font-size: 15px;
+  text-decoration: none;
+  display: flex;
+  justify-content: space-between;
+  font-weight: 500;
+}
+
+#ext {
+  height: 24px;
+  position: absolute;
+  margin-left: -55px;
+  margin-top: 11px;
+}
+
+#g {
+  position: absolute;
+  margin-top: 13px;
+  margin-left: -30px;
+}
+
+#pop-x {
+  margin-top: 16px;
+  margin-left: 117px;
+  position: absolute;
+  filter: brightness(0) invert(1);
+}
+
+.pop2 {
+  display: none;
+}
+
+.skills {
+  display: flex;
+  margin-top: 60px;
+}
+
+.skill {
+  width: 340px;
+  height: 345px;
+  border: 1px solid #c1c7d0;
+  border-radius: 8px;
+  background-color: #ebf0ee;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 12px;
+  line-height: 16px;
+}
+

```

Comment:
- [ ] Please make sure, in the mobile version, the close button is white as it's in the figma design. 

**Your project design**
![image](https://user-images.githubusercontent.com/74411135/233714593-e2195709-213a-446d-85ea-9c2e20305238.png)
 
**figma design**
![image](https://user-images.githubusercontent.com/74411135/233714704-c21ed940-89e7-4844-8adb-8d2cd304da19.png)

This will make your project prettier and make the user know you're an awesome developer as you can precisely follow a professional web developer's design 😃 💪